ccmlib/cluster.py

```diff
@@ -247,8 +247,16 @@ def add(self, node, is_seed, data_center=None):
         for trace_class in self._trace:
             node.set_log_level("TRACE", trace_class)
 
+        if data_center is None:
+            for existingNode in self.nodelist():
```

Comment:
naming should be `existing_node` to not break flake8 rules.

Reply:
ha! good catch. My java brain kicking in lol...